src/main/java/org/openjdk/jextract/impl/HeaderFileBuilder.java

```diff
@@ -238,6 +245,18 @@ void emitPointerTypedef(Declaration.Typedef typedefTree, String name) {
         emitPrimitiveTypedefLayout(name, TypeImpl.PointerImpl.POINTER_LAYOUT, typedefTree);
     }
 
+    void emitLoadLibraries(List<String> libraries) {
```

Comment:
Right, I was thinking of dropping it, but I've kept it for now, since we also fall back to the default lookup, and I'm not sure how important that is.